### PR TITLE
Data viz

### LINKED ISSUE #15: [Data] Add code for generating maps of bus routes and other data visualizations

### PR DESCRIPTION
**Pull request by dcjohnson24**
# Description
This pull request is to add code for generating maps of bus routes colored by the ratio of actual trips to scheduled trips. It also contains plots generated from the code. This will be a starting point for generating future maps for other variables of interest.

Resolves #15 

## Type of change

- [ ] Bug fix
- [x] New functionality
- [ ] Documentation

## How has this been tested?
This has been tested on a local copy of the repository.

### CODE REVIEW

File: data_analysis/plots.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+       of completed trips
+
+    Args:
+        df (Union[pd.DataFrame, gpd.GeoDataFrame]): DataFrame with ratio
+            of completed trips by route_id
+        n (int, optional): number of route_ids to return . Defaults to 10.
+        lowest (bool, optional): whether to return the lowest ratios
+    Returns:
+        Union[pd.DataFrame, gpd.GeoDataFrame]: A DataFrame containing the
+            n worst routes.
```
Comment (lauriemerrell) [inline data_analysis/plots.py:37]: nitpick: I think this could be n worst or n best?

File: data_analysis/plots.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+import geopandas as gpd
+import pandas as pd
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import compare_scheduled_and_rt
+import static_gtfs_analysis
+
+CHICAGO_COORDINATES = (41.85, -87.68)
+PLOTS_PATH = Path(__file__).parent.parent / 'plots' / 'scratch'
```
Comment (lauriemerrell) [inline data_analysis/plots.py:18]: I wonder if we should make the git repo root path an environment variable and then use that to construct these paths, just to make it more generalizable?

**Review by lauriemerrell — APPROVED** [review]
soooooo excited about this! two inline small comments, neither one blocking

File: data_analysis/plots.py (hunk 1 above)
Comment (dcjohnson24) [reply data_analysis/plots.py:37]: What about something like `n_worst_best_routes`?

File: data_analysis/plots.py (hunk 2 above)
Comment (dcjohnson24) [reply data_analysis/plots.py:18]: That would probably be a better bet. I'll update this.

**Review by lauriemerrell — APPROVED** [review]
LGTM, thank you!!

**Pull request merged**